auth_oauth_multi_token/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+OAuth multi token
+=================
+
```

Comment:
please use the new fragment [readme](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0)

Reply:
Predo says is not mandatory.

Reply:
@fmdl are you sure? It's in the migration guidelines. You dont have to create the readme, as it's generated from the fragment rst files, but you have no fragment files.